client/interop/musicxml/importer/util.go

```diff
@@ -330,3 +336,68 @@ func isOrContainsBarline(update model.ScoreUpdate) bool {
 
 	return false
 }
+
+// idiomaticDuration applies simple heuristics to make durations more idiomatic
+func idiomaticDuration(aldaDuration float64, dots int32) model.Duration {
```

Comment:
Nice work on this part, in particular. This looks like it was hard to write, but it's totally worthwhile!